cmd/micro/bindings.go

```diff
@@ -255,6 +255,7 @@ var bindingKeys = map[string]tcell.Key{
 	"Escape":         tcell.KeyEscape,
 	"Enter":          tcell.KeyEnter,
 	"Backspace":      tcell.KeyBackspace2,
+	"OldBackspace":   tcell.KeyBackspace,
```

Comment:
turns out, mate-terminal sends CtrlBackspace as tcell.KeyBackspace, and there wasn't another way to bind to that behavior, since everything explicitly referenced tcell.KeyBackspace2.